src/util.cpp

```diff
@@ -936,3 +927,10 @@ int64_t GetStartupTime()
 {
     return nStartupTime;
 }
+
+fs::path NormalizePathArg(const fs::path& pathArg, bool fNetSpecific)
```

Comment:
I'd rename to `GetAbsolutePath` or more specific `GetAbsoluteDataPath`:
```cpp
fs::path GetAbsoluteDataPath(const fs::path& path, bool net_specific)
```
I other contexts/languages, normalization can be interpreted as transforming from `foo/../bar/` to `bar`.

Nit, snake_case arguments.